python/ribasim/ribasim/geometry/edge.py

```diff
@@ -42,10 +42,6 @@ class Config:
 class EdgeTable(SpatialTableModel[EdgeSchema]):
     """Defines the connections between nodes."""
 
-    def __init__(self, **kwargs):
-        kwargs.setdefault("df", DataFrame[EdgeSchema]())
-        super().__init__(**kwargs)
```

Comment:
This is needed.

Ribasim Core always requires an Edge table, even if empty.
Without this, no Edge table is added if you don't have edges

Reply:
Yep, but this interferes with the default Pydantic initialization, I fixed it in another way.